plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcClient.java

```diff
@@ -216,4 +217,6 @@ default Optional<TableScanRedirectApplicationResult> getTableScanRedirection(Con
     OptionalLong delete(ConnectorSession session, JdbcTableHandle handle);
 
     void truncateTable(ConnectorSession session, JdbcTableHandle handle);
+
+    OptionalInt getMaxWriteParallelism(ConnectorSession session);
```

Comment:
Is there any reason to expose this method in `JdbcClient`?

Reply:
Removed.